webpage/jupyter_notebook_jekyll_theme/_sass/_navigation.scss

```diff
@@ -175,7 +175,7 @@
 .greedy-nav {
   position: relative;
   min-width: 250px;
-  background-color: $greedy-nav-background-color;
+  background-color: #5E5C57;
```

Comment:
Same, not changing in https://github.com/BiG-CZ/notebook_data_demo/blob/master/webpage/jupyter_notebook_jekyll_theme/_sass/_variables.scss#L66?